### PR TITLE
Bugfixes/migration bugs

### PR DESCRIPTION
**Pull request by georgend**
Script Bugfixes due to migration to new tenant. 

Add data scripts for AccessPackage entities where permissions are available. 
Move known permission issues to Known Issues list. 
Raise issues in related repos for known issues and emerging issues.

### CODE REVIEW

File: scripts/TenantSetup/RaptorUtils.ps1 [hunk 1]
```diff
@@ -188,6 +188,11 @@ function Get-Scopes
         [Parameter(Mandatory = $True)][string] $Path
     )
     $scopes = @()
+    if ($Path[0] -ne "/") # DevX API expects a trailing slash
```
Comment (zengin) [inline scripts/TenantSetup/RaptorUtils.ps1:191]: fixing my typo :)
```suggestion
    if ($Path[0] -ne "/") # DevX API expects a leading slash
```

File: azure-pipelines/typescript-beta-compile-tests.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
```
Comment (zengin) [inline azure-pipelines/typescript-beta-compile-tests.yml:1]: please remove this file. Let's not merge two different sets of work in a single PR ;)

**Review by zengin — CHANGES_REQUESTED** [review]

File: azure-pipelines/typescript-beta-compile-tests.yml (hunk 2 above)
Comment (georgend) [reply azure-pipelines/typescript-beta-compile-tests.yml:1]: My bad. Purging

**Review by fey101 — APPROVED** [review]
LGTM

**Review by zengin — APPROVED** [review]

**Pull request merged**